tests/test_handlers.py

```diff
@@ -824,8 +824,6 @@ async def test_get_folders_with_bad_params(self):
 
     async def test_get_folder_works(self):
         """Test folder is returned when correct folder id is given."""
-        RESTAPIHandler._handle_check_ownership = make_mocked_coro(True)
```

Comment:
removed a duplicate row that seemed to do nothing, the function is already mocked in the test case setup https://github.com/CSCfi/metadata-submitter/blob/feature/project-ownership/tests/test_handlers.py#L103